data/default_release_images.json

```diff
@@ -37,15 +37,15 @@
     },
     {
         "openshift_version": "4.11",
-        "cpu_architecture": "x86_64",
-        "url": "quay.io/openshift-release-dev/ocp-release:4.11.0-x86_64",
+        "cpu_architecture": "multi",
+        "cpu_architectures": [
```

Comment:
so what is the intended difference between ``cpu_architecture`` and ``cpu_architectures``?
maybe it's better to rename ``cpu_architecture`` to ``image_architecture`` to annotate it as the one relevant to the docker image arch type?

Reply:
Both fields have the same meaning. One is `string`, the other is `[]string` but the underlying meaning is the same for both of them. The naming (i.e. `cpu_architectures`) has been agreed on https://github.com/openshift/assisted-service/pull/4248. In the future we will deprecate `string` in favour of `[]string`, but now we need to keep backwards compatibility